spec/features/external_request_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature 'External request' do
+  it 'queries FactoryGirl contributors on GitHub' do
```

Comment:
What is this ? This should not be here I think?

Reply:
Let's clean up the tests in the next feature branch, okay?